### PR TITLE
model: add new field to explict charset&collation in ColumnInfo

### PR DESCRIPTION
**Pull request by xiekeyi98**
### What problem does this PR solve? 
To solve https://github.com/pingcap/tidb/pull/9113 , we need change ColumnInfo a little.

### What is changed and how it works?

Add a field `ExplictedCollation` .

### CODE REVIEW

File: model/model.go [hunk 1]
```diff
@@ -83,6 +83,7 @@ type ColumnInfo struct {
 	types.FieldType     `json:"type"`
 	State               SchemaState `json:"state"`
 	Comment             string      `json:"comment"`
+	ExplictedCollation  bool        `json:"explicted_collation"`
```
Comment (xiekeyi98) [inline model/model.go]: Any other **better variable name** ? I think this name is not good enough, but I cannot find a better name(avoid too long).
And I modeled the variable name of MYSQL(In MYSQL, this name is `column_has_explicit_collation`).

Comment (kennytm) [inline model/model.go]: "Explicit" is an adjective not a verb, there's no need to add the -ed suffix.

```suggestion
	ExplicitCollation   bool        `json:"explicit_collation"`
```

Comment (xiekeyi98) [reply model/model.go]: Done,thanks for your help.

**Comment by xiekeyi98** [comment]
I make something wrong.
Sorry for bother everyone.
This PR will be closed because it is not necessary now.

**Pull request closed without merging**